src/models/task/postgres.js

```diff
@@ -59,10 +60,32 @@ const TaskRepo = (postgres) => {
         }
     };
 
+    const setTaskCompleteSQL = `
+        UPDATE tasks SET completed=true, timeCompleted=CURRENT_TIMESTAMP
+        WHERE id=$1
+        RETURNING *;
+    `
+    //my work
+    const setTaskCompleted = async (taskID) => {
+        const values = [taskID];
+        try {
+            const client = await postgres.connect(); //client is an object that has a connection to DB 
+            const res = await client.query(setTaskCompleteSQL, values);
+            client.release();
+            return [res.rows[0], null]; //requires null because no error in this case
+        }
+        catch (err) {
+            return [null, err]; // return null for the actual object, the error should not be null
```

Comment:
Similar to above, instead of returning `null`, I think I had it return an empty object.